src/sentry/utils/math.py

```diff
@@ -30,3 +30,33 @@ def mad(values, K=1.4826):
     # http://en.wikipedia.org/wiki/Median_absolute_deviation
     med = median(values)
     return K * median([abs(val - med) for val in values])
+
+
+def nice_int(x):
+    """
+    Round up to the nearest "nice" number.
+    """
+
+    if x == 0:
```

Comment:
Probably an edge case 🤷 
```suggestion
    if x < 0:
```

Reply:
I thought about this, but returning 0 here seems counter intuitive. Would rather have this function error for negative numbers or  perhaps apply the same transformation to the absolute value and reapply the sign.

Reply:
Hmm i think we should either `assert x>0` and raise a msg explaining that we require x >= 0 or do your suggestion of using the absolute value. Since we'll get a `math domain error` on the `log10` line